### PR TITLE
Add flyto

### PR DESCRIPTION
**Pull request by seddonm1**
This pull request exposes NativeMapView.flyTo

### CODE REVIEW

File: lib/mapbox_map.dart [hunk 1]
```diff
@@ -115,6 +105,25 @@ class MapboxMap  {
     }
   }
 
+  Future<Null> flyTo(double angle, LatLng center, int duration, double pitch,
```
[the remark this answers is not in the document]
Comment (tobrun) [reply lib/mapbox_map.dart]: Can we change the public API signature to CameraPosition instead of all fields separately?

**Comment by seddonm1** [comment]
i think we can do this and #13 super quickly once we solve #21

**Comment by seddonm1** [comment]
@tobrun I think this method solves both #21, #18 and #13 .... although I don't understand difference between flyTo and easeTo (why is easingInterpolator boolean?)

Try the demo as it is quite a full example

**Comment by tobrun** [comment]
> I don't understand difference between flyTo and easeTo (why is easingInterpolator boolean?)

flyTo performs a hyperbolic flight animation (changes zoom) while ease maintains the same zoom level while animating. The ease boolean is a left over from an old api and allows to use a linear interpolator instead of an easing interpolator.

**Comment by tobrun** [comment]
Landed with 6400175, thanks again for the contirbution @seddonm1!

**Pull request merged**